### PR TITLE
Feature: cookie based auth

### PR DESCRIPTION
**Pull request by ZakariaTalhami**
Changes:
* Remove tokens from the response body of login API
* Set "Set-Cookie" header for access and refresh token cookies

### CODE REVIEW

**Comment by ZakariaTalhami** [comment]
Is anyone aware of a better way to implement setting cookies in the header of the response?

File: backend/views/users.py
Comment (believemanasseh) [inline backend/views/users.py:1]: Seems you're assigning the access token to both the access_token and refresh_token cookies. Also, for the registration route, I guess 201 is the right HTTP status code.

**Comment by believemanasseh** [comment]
I'm not sure this change is needed again. The cookie-based authentication is being implemented with redis

**Comment by zatoa** [comment]
It could be a good idea to encrypt the cookie, a lot of auth libs do this, even if set to httponly

**Comment by MrBisquit** [comment]
> It could be a good idea to encrypt the cookie, a lot of auth libs do this, even if set to httponly

If it's JWT there's no need for it

**Pull request closed without merging**